tests/test_reuse.py

```diff
@@ -40,33 +40,32 @@ def set_up_repo(tmp_path, template_path, template_name, license_path, license_na
     # Change dir to tmp_path
     os.chdir(tmp_path)
 
+    # Make asset directories if using a custom license or template
+    # Asset directories are .reuse and LICENSES
+    make_asset_dirs(tmp_path, template_path, template_name, license_path, license_name)
+
     # Set up git repository in tmp_path
-    repo = init_git_repo(tmp_path, template_path, template_name, license_path, license_name)
+    git.Repo.init(tmp_path)
+    repo = git.Repo(tmp_path)
+    repo.index.commit("initialized git repo for tmp_path")
 
     # Create a test file in tmp_path
     tmp_file = create_test_file(tmp_path)
 
     return repo, tmp_file
 
 
-def init_git_repo(tmp_path, template_path, template_name, license_path, license_name):
-    """Initialize git repository and add the .reuse directory & template."""
-    reuse_dir = os.path.join(tmp_path, ".reuse", "templates")
-    license_dir = os.path.join(tmp_path, "LICENSES")
-
-    # Create .reuse directory and copy template file to it
-    os.makedirs(reuse_dir)
-    shutil.copyfile(template_path, f"{reuse_dir}/{template_name}")
-
-    os.makedirs(license_dir)
-    shutil.copyfile(license_path, f"{license_dir}/{license_name}")
-
-    # Initialize tmp_path as a git repository
-    git.Repo.init(tmp_path)
-    repo = git.Repo(tmp_path)
-    repo.index.commit("initialized git repo for tmp_path")
+def make_asset_dirs(tmp_path, template_path, template_name, license_path, license_name):
```

Comment:
Interesting as well.. why did you have to add this function inside the test module?

Reply:
I had to add that function for the tests that had custom templates and/or licenses because the user should have these folders in their repo if they are using anything custom